src/components/Header/index.js

```diff
@@ -28,6 +28,9 @@ function Header() {
         <Link to="/blog" color="neutral-4">
           Blog
         </Link>
+        <Link to="/search" color="neutral-4">
```

Comment:
At first, I thought this is React Link component, so the name appeared as a little confusing. It's good to Link is using Anchor under the hood to leverage the theming aspect.
I looked a little bit on how Link is implemented, quick comment there, `children` isn't needed if you are using `...rest`
```
const Link = ({ children, to, ...rest }) => (
  <Anchor
    href={to}
    onClick={ev => {
      navigate(to);
      ev.preventDefault();
    }}
    {...rest}
  >
    {children}
  </Anchor>
);
```

Reply:
Also, line #18 will be a good candidate to use the new Header component here (will be released soon)

Reply:
Cool. Thanks. Changed.